src/Symfony/Component/Asset/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.0
+---
+
+* removed `RemoteJsonManifestVersionStrategy`, use `JsonManifestVersionStrategy` instead.
```

Comment:
`remove` and without final `.`
https://symfony.com/doc/current/contributing/code/conventions.html#writing-a-changelog-entry

Reply:
Updated